api/controllers/UserController.js

```diff
@@ -63,6 +63,30 @@ module.exports = {
       }
       // Query for game if user is in one
       const gameId = (user.game ?? req.session.game) ?? null;
+
+      if (process.env.VITE_USE_GAMESTATE_API) {
+        const { unpackGamestate, createSocketEvent } = sails.helpers.gameStates;
+        const game = await Game.findOne(gameId).populate('gameStates')
+          .populate('p0')
+          .populate('p1');
+        const gameObject = await unpackGamestate(game.gameStates.at(-1));
```

Comment:
We need to populate the gameStates for them to be available for the createSocketEvent call. Additionally we should handle the case where there are no gameStates because you're in the lobby

Reply:
It's being populated on line 69, or do we need to do something different?

Reply:
Ah silly me I just missed it because the other two were on new lines and I reviewed on my phone. This is right. The only thing is we should handle the case where there are no gamestates yet ie toy refresh from the lobby

Reply:
:) working on that at the moment